middleware/home-preview.js

```diff
@@ -0,0 +1,7 @@
+export default function ({ route, redirect }) {
+  // Contentful uses '/' as a slug for the home page, and iterpolates
```

Comment:
"interpolates"

Reply:
Also, reword slightly as it's not Contentful that uses it, but our content model in Contentful.